packages/utils/theme/src/defaultThemeDark.ts

```diff
@@ -3,6 +3,18 @@ import { type Theme } from './types'
 
 export const defaultThemeDark: Theme = createTheme({
   colors: {
+    // Basic
+    basic: '#000000', // missing in figma as of now
+    onBasic: '#1D1F2A',
+    basicContainer: '#4D4E7B',
+    onBasicContainer: '#FFFFFF',
+    // Accent
+    accent: '#CC99FF',
+    onAccent: '#1D1F2A',
+    accentContainer: '#8D64BB',
+    onAccentContainer: '#FFFFFF',
+    accentVariant: '#DBB7FF',
+    onAccentVariant: '#1D1F58',
```

Comment:
How do you know that "accent" family has "accentVariant", but "basic family has no "basicVariant" (and so on). Is there already a spec somewhere ?

Reply:
I used this commit as a reference 👉 https://github.com/adevinta/spark-tokens/pull/56/commits/73133e8cf67ce246da382c8cb00a5becec31a885

Reply:
I will update my PR in the morning and base myself on the Figma file instead 😅